mcstatus/bedrock_status.py

```diff
@@ -80,6 +82,16 @@ def __init__(self, protocol: int, brand: str, version: str):
             self.brand = brand
             self.version = version
 
+    class Motd(str):
+        """Just add method `clean` to `motd` field."""
+
+        def clean(self) -> str:
+            """Remove colors (&7, &a, §1 etc.) from description.
+
+            :return: Clean description, without colors.
+            """
+            return re.sub(FIND_MINECRAFT_COLORS_REGEX, "", self)
+
```

Comment:
It may be worth it to make `Description` class in `pinger.py` global, and just import it here directly. I'm not a huge fan of doing the same thing multiple times in different places.

This will become a bit easier after #306 is merged, providing a common place for this class to be in. But right now, let's not make 2 different classes with same logic for the same thing.

Reply:
Removed `Motd` class and using `PingResponse.Descriprion` instead now.